cmd/vulcan-exposed-http-resources/main.go

```diff
@@ -234,6 +289,7 @@ func checkResource(targetURL *url.URL, httpResource Resource) []map[string]strin
 					return
 				}
 
+				// This line creates a copy of the net.URL object.
 				targetResource, err := url.Parse(targetURL.String())
```

Comment:
Not a big deal, but you don't need to parse the url again to make a copy, you can do something like this:
https://play.golang.org/p/H-2gGbNwKsh
But to be honest there is no difference in terms of performance or anything.

Reply:
I was not aware of this, it is a good thing to know for the future! In this case, I think I will leave it like it is since it was not introduced in this PR and I like that it takes a single line that is easily understandable. Thanks a lot for the tip!